netcdf/const.go

```diff
@@ -63,3 +63,7 @@ var typeNames = map[Type]string{
 	UINT64: "UINT64",
 	STRING: "STRING",
 }
+
+func (t Type) Name() string {
```

Comment:
This should be called String(), so that fmt package knows how to convert to a string.